src/index.json

```diff
@@ -298,6 +298,50 @@
                 "sha256Digest": "1faecf9d3b5955829e00b228e174fa6bf1ba327f0d64c2edfc9d096d2926e8dc"
             }
         ],
+        "azure-devops": [
+            {
+                "downloadUrl": "https://azuredevopscli.blob.core.windows.net/releases/azure_devops-1.0.0rc1-py2.py3-none-any.whl",
+                "filename": "azure_devops-1.0.0rc1-py2.py3-none-any.whl",
+                "metadata": {
+                    "azext.minCliCoreVersion": "2.0.49",
+                    "azext.isPreview": true,
+                    "extensions": {
+                        "python.details": {
+                            "contacts": [
+                                {
+                                    "email": "VSTS_Social@microsoft.com",
+                                    "name": "Microsoft",
+                                    "role": "author"
+                                }
+                            ],
+                            "document_names": {
+                                "description": "DESCRIPTION.rst"
+                            },
+                            "project_urls": {
+                                "Home": "https://github.com/Microsoft/vsts-cli/tree/azuredevopscli-dev"
+                            }
+                        }
+                    },
+                    "extras": [],
+                    "generator": "bdist_wheel (0.30.0)",
+                    "license": "MIT",
+                    "metadata_version": "2.0",
+                    "name": "azure-devops",
+                    "run_requires": [
+                        {
+                            "requires": [
+                                    "keyring (~=13.2.1)",
+                                    "python-dateutil (==2.7.3)",
+                                    "vsts (==0.1.20)"
+                            ]
+                        }
+                    ],
+                    "summary": "Tools for managing Azure DevOps.",
+                    "version": "1.0.0rc1"
```

Comment:
Leave this as 0.1.0 as this is still preview.

Reply:
@williexu : made the requested changes.
The PR build is failing and I am not able to understand the reason, can you please help with debugging the failure?
https://travis-ci.org/Azure/azure-cli-extensions/jobs/471761456

Reply:
@williexu : gentle ping

Reply:
@gauravsaralMs there are problems with your extension that is causing the CI to fail.
I'd recommend:
1. doing a fresh dev install(which will give you access to `azdev`) of the CLI through: [this](https://github.com/Azure/azure-cli/blob/dev/doc/configuring_your_machine.md)
2. Installing your extension: `az extension add -s https://azuredevopscli.blob.core.windows.net/releases/azure_devops-0.1.0-py2.py3-none-any.whl`
3. Running: `azdev cli-lint --ci --extensions azure-devops` and debugging to fix all errors you see.

Reply:
@williexu Thanks for looking into it
We haven't specified any maxCliCoreVersion as of now so hopefully I don't need to update anything..
Query: Once your PR https://github.com/Azure/azure-cli/pull/8127 is merged then CI issue will be resolved?  (I read your comment but confirming again)

Reply:
@gauravsaralMs oops sorry I meant **min** cli core version. My bad.